### PR TITLE
Model-driven-operations build

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Built new page as per [copy doc](https://docs.google.com/document/d/1G4sSrM-FQIL2c1Uj8PvW6oGUKcJ9HAK5TwnKhFa4sOY/edit#)
- Added page to #enterprise nav
- Added page to #developer nav
- Added page to footer
## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-11026.demos.haus/model-driven-operations
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against copy doc and see all changes have bee made
- Test nav and footer links and see that they all work as expected

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4679

### CODE REVIEW

File: templates/model-driven-operations/index.html [hunk 1]
```diff
@@ -0,0 +1,473 @@
...
+      <p>JAAS is the best way to quickly model and deploy your cloud-based applications. Create, configure and host your applications on the public cloud directly from your browser or the command line.  Concentrate on your software and solutions with a fully managed Juju infrastructure.</p>
+      <p>
+        <a class="p-button--positive p-link--external" href="https://jaas.ai/">Get started with JAAS</a>
+      </p>
+    </div>
+    <div class="col-6">
+      <div class="u-hide--small u-align--center u-vertically-center">
+        {{ 
+          image (
+          url="https://assets.ubuntu.com/v1/1dfbab48-jaas.gif",
```
Comment (mmanciop) [inline templates/model-driven-operations/index.html:384]: This does not work in Firefox on Linux. I just see the `Alt`.

File: templates/model-driven-operations/index.html [hunk 2]
```diff
@@ -0,0 +1,473 @@
...
+          alt="Jaas gif",
```
Comment (mmanciop) [inline templates/model-driven-operations/index.html:385]: Please change to "Model-driven operations in JAAS"

File: templates/model-driven-operations/index.html [hunk 3]
```diff
@@ -0,0 +1,473 @@
...
+        }}
+      </div>
+      <p>Juju Community: Join the conversation on <a class="p-link--external" href="https://discourse.charmhub.io/">Discourse</a> and <a class="p-link--external" href="https://chat.charmhub.io">Mattermost</a>. Meet the team at the “<a class="p-link--external" href="https://discourse.charmhub.io/t/community-events/5295/8">Charmed Office Hours</a>”. Anyone and everyone is welcome to join!</p>
+    </div>
+    <div class="col-3 p-divider__block">
+      <div class="u-hide--small u-align--center">
+        {{ 
+          image (
+          url="https://assets.ubuntu.com/v1/86fa95f9-icon-orange-blog.svg",
+          alt="Blog icon",
```
Comment (mmanciop) [inline templates/model-driven-operations/index.html:439]: Please change to "Juju blogs"

File: templates/model-driven-operations/index.html [hunk 4]
```diff
@@ -0,0 +1,473 @@
...
+        }}
+      </div>
+      <p>Juju <a href="https://juju.is/blog">Blogs</a>: All the latest news on Juju – the open-source application modelling tool for public and private clouds. Start reading the <a class="p-link--external" href="https://juju.is/blog/model-driven-observability-part-1">post on model-driven observability</a>.</p>
+    </div>
+    <div class="col-3 p-divider__block">
+      <div class="u-hide--small u-align--center u-sv3">
+        {{ 
+          image (
+          url="https://assets.ubuntu.com/v1/21910613-kubernetes-orangebox.svg",
+          alt="Kubernetes orangebox",
```
Comment (mmanciop) [inline templates/model-driven-operations/index.html:454]: Please change to "Juju for the cloud-native community"

**Review by mmanciop — CHANGES_REQUESTED** [review]
LGTM except for a few things, annotated in the PR.

Also, QA-ing around, I noticed we should also change this link to point to this new page: https://github.com/canonical-web-and-design/ubuntu.com/blob/dbf8a87ac11a5bc276b2e1d8d7770f01a8db0bff/templates/templates/navigation-enterprise-h.html#L157 . Could you please add it to this PR?

File: templates/model-driven-operations/index.html [hunk 5]
```diff
@@ -0,0 +1,473 @@
...
+    <div class="col-7">
+      <h1>Model-driven operations: the best way to run enterprise software</h1>
+      <p>Modern software is complex and expensive to run reliably at scale. The setups are complicated and hard to maintain. Time to put structure in the chaos!</p>
+      <p>Model-driven operations let you save effort and focus by declaratively composing your applications out of reusable, proven and flexible operators and running them smoothly and toil-free.</p>
+    </div>
+    <div class="col-5 u-hide--small u-align--center u-vertically-center">
+      {{ 
+        image (
+        url="https://assets.ubuntu.com/v1/2c7e3fab-juju-header-illustration.svg",
+        alt="Juju",
```
Comment (mmanciop) [inline templates/model-driven-operations/index.html:22]: Please change the `alt` to "Model-driven operations with Juju"

File: templates/model-driven-operations/index.html [hunk 6]
```diff
@@ -0,0 +1,473 @@
...
+        <a href="/openstack/managed">Openstack</a>
+      </p>
+      <p>A fully integrated and optimised combination of the latest release of Ubuntu Server and the latest release of OpenStack.</p>
+    </div>
+    <div class="col-4">
+      <div class="u-hide--small u-align--center">
+        {{ 
+          image (
+          url="https://assets.ubuntu.com/v1/e68fd355-1+discovery_AW.svg",
+          alt="Observability icon",
```
Comment (mmanciop) [inline templates/model-driven-operations/index.html:181]: Please change the `alt` to "Best-of-breed open-source observability with Juju"

File: templates/model-driven-operations/index.html [hunk 7]
```diff
@@ -0,0 +1,473 @@
...
+        <a href="/kubernetes/managed">Kubernetes</a>
+      </p>
+      <p>A curated set of tools to help you build your Kubernetes clusters from the ground up, and deliver ‘containers as a service’ across the enterprise.</p>
+    </div>
+    <div class="col-4">
+      <div class="u-hide--small u-align--center">
+        {{ 
+          image (
+          url="https://assets.ubuntu.com/v1/95abe343-image-project-openstack.png",
+          alt="Openstack",
```
Comment (mmanciop) [inline templates/model-driven-operations/index.html:163]: Please change the `alt` to "Charmed OpenStack with Juju is the best way to run OpenSTack"

File: templates/model-driven-operations/index.html [hunk 8]
```diff
@@ -0,0 +1,473 @@
...
+      <h2 class="u-sv4 u-align--center">A vibrant ecosystem, one deploy command away</h2>
+    </div>
+  </div>
+  <div class="row u-equal-height u-sv4">
+    <div class="col-4">
+      <div class="u-hide--small u-align--center">
+        {{ 
+          image (
+          url="https://assets.ubuntu.com/v1/dba11aee-kubernetes.svg",
+          alt="Kubernetes",
```
Comment (mmanciop) [inline templates/model-driven-operations/index.html:145]: Please change the `alt` to "Efficient, portable Kubernetes with Juju"

File: templates/model-driven-operations/index.html [hunk 9]
```diff
@@ -0,0 +1,473 @@
...
+      </div>
+      <h3>Bare metal and virtual machines</h3>
+      <p>Provide to your users a cloud-like experience while retaining full ownership over your data centres.</p>
+    </div>
+    <div class="col-4">
+      <div class="u-hide--small u-align--center u-sv2">
+        {{ 
+          image (
+          url="https://assets.ubuntu.com/v1/fece8a11-container-in-the-cloud.svg",
+          alt="Container",
```
Comment (mmanciop) [inline templates/model-driven-operations/index.html:117]: Please change the `alt` to "Juju can operate your cloud-native applications and containers"

File: templates/model-driven-operations/index.html [hunk 10]
```diff
@@ -0,0 +1,473 @@
...
+      </div>
+      <h3>Public clouds and Multi-cloud</h3>
+      <p>Combine the advantages of public cloud services while enjoying long-term cost savings, increased flexibility and reduced vendor lock-in.</p>
+    </div>
+    <div class="col-4 ">
+      <div class="u-hide--small u-align--center u-sv2">
+        {{ 
+          image (
+          url="https://assets.ubuntu.com/v1/9ed50294-Virtual+machine.svg",
+          alt="Virtual machine",
```
Comment (mmanciop) [inline templates/model-driven-operations/index.html:101]: Please change the `alt` to "Juju manages Virtual Machines running on various hypervisors"

File: templates/model-driven-operations/index.html (hunk 1 above)
Comment (mmanciop) [reply templates/model-driven-operations/index.html:384]: Interestingly, this URL seems to work just fine in Firefox mmm https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_1280,h_720/https://assets.ubuntu.com/v1/1dfbab48-jaas.gif

Comment (mtruj013) [reply templates/model-driven-operations/index.html:384]: Its working on the demo now too!

**Review by mmanciop — COMMENTED** [review]
Good stuff! We need to solve that vertical  alignment issue of the images in the “Get started today!” section and this first iteration is IMO good to ship.

File: templates/model-driven-operations/index.html [hunk 11]
```diff
@@ -0,0 +1,473 @@
...
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
```
Comment (minkyngkm) [inline templates/model-driven-operations/index.html:35]: image

Would it be better if these 3 sections have `col-8` as they have no images on the side? What do you think @spencerbygraves?

File: templates/model-driven-operations/index.html [hunk 12]
```diff
@@ -0,0 +1,473 @@
...
+
+<hr class="is-fixed-width">
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
+      <h2>JAAS: Managed Juju infrastructure, hosted by Canonical</h2>
+      <p>JAAS is the best way to quickly model and deploy your cloud-based applications. Create, configure and host your applications on the public cloud directly from your browser or the command line.  Concentrate on your software and solutions with a fully managed Juju infrastructure.</p>
+      <p>
+        <a class="p-button--positive p-link--external" href="https://jaas.ai/">Get started with JAAS</a>
```
Comment (minkyngkm) [inline templates/model-driven-operations/index.html:377]:
```suggestion
        <a class="p-button--positive" href="https://jaas.ai/">Get started with JAAS</a>
```

**Comment by davegoddard42** [comment]
@spencerbygraves could you take a look when you get a chance? Thanks!

**Comment by spencerbygraves** [comment]
Thanks @mtruj013 a few things from me please:

- Can you remove `hr-fixed-width` where shown please, and make the "On rails" section `p-strip--light` please
- We don't use centred text, so can you make all headings left aligned as shown please

Thanks

image

**Review by mmanciop — APPROVED** [review]

**Review by spencerbygraves — APPROVED** [review]

File: templates/model-driven-operations/index.html [hunk 13]
```diff
@@ -0,0 +1,460 @@
+{% extends "model-driven-operations/_base_model.html" %}
+
+{% block title %}Model-driven operations: the best way to run enterprise software{% endblock %}
+
+{% block meta_description %}Save effort and regain focus through model-driven operations by declaratively composing your applications out of reusable, proven and flexible operators.{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1G4sSrM-FQIL2c1Uj8PvW6oGUKcJ9HAK5TwnKhFa4sOY/edit#heading=h.kpn77t6yr8it{% endblock meta_copydoc %}
```
Comment (minkyngkm) [inline templates/model-driven-operations/index.html:7]:
```suggestion
{% block meta_copydoc %}https://docs.google.com/document/d/1G4sSrM-FQIL2c1Uj8PvW6oGUKcJ9HAK5TwnKhFa4sOY/edit#{% endblock meta_copydoc %}
```

**Review by minkyngkm — APPROVED** [review]
LGTM @mtruj013! 👍  Just check the tag id please :)

**Pull request merged**